zulip_bots/zulip_bots/bots/wikipedia/doc.md

```diff
@@ -0,0 +1,13 @@
+# Wikipedia bot
+
+## Intro
+
+This bot returns the top wikipedia link for a search term entered by the user.
+
+## Setup
+
+Instructions for running this bot can be found [here](https://zulipchat.com/api/running-bots#running-a-bot).
+
+## Usage
+
+Use '@mention-bot' at the start of your message and enter a key term afterwards to request an article.

```

Comment:
You could expand this a bit showing alternative usages. If you look into this bot's tests (https://github.com/zulip/python-zulip-api/blob/master/zulip_bots/zulip_bots/bots/wikipedia/test_wikipedia.py), it can handle different inputs.
For example, if the input is a number, the bot returns the article with id <number> instead of a wiki page of that number.
So show examples of query include single word, multiple words, number and hash.